### ISSUE TITLE
Expose `ignore_missing_samples` parameter in visualizers?

### ISSUE THREAD

**Issue by thermokarst** [qiime2/q2-emperor#64]
This issue proposes the addition of a new parameter (`ignore_missing_samples`) to the visualizers defined in this plugin.

**Current Behavior**
Currently, the visualizations in this plugin (`plot` and `procrustes_plot`) perform [index matching](https://dev.qiime2.org/latest/glossary/#term-identifier) of the Sample IDs in the PCoA results and the Metadata. This index matching is conducted entirely by `emperor` (rather than q2-emperor). When a user provides metadata to these actions that are missing samples that _are_ present in the PCoA results, `emperor` will raise the following error:

```bash
There are samples not included in the mapping file.
Override this error by using the ignore_missing_samples argument.
```

**Proposed Behavior**
Emperor supports an optional parameter called `ignore_missing_samples` that will disable or enable index matching between the PCoA results and metadata at runtime. This parameter could be exposed in both `plot` and `procrustes_plot` as a `Visualizer` parameter. In the case where there are missing samples in the metadata, _and_ `ignore_missing_samples` is `True`, the error above would not be raised, and instead `emperor` would just strip away those metadata-less samples from the plot.

**Questions**
1. There is currently no index matching in the plugin itself, which means that users will *only* see `emperor`-derived error messages related to ID mismatches, which is probably an inconsistent UX for users. I raise the question here, since they are somewhat related, but would be equally happy addressing in another issue.
2. If this parameter is added, should it be exposed in the q2-diversity `core-metrics*` pipelines?

**Comments**
1. There is precedent for this kind of per-action ID matching within the QIIME 2 ecosystem (see link below RE: q2-sample-classifier).

**References**
1. [Originating forum post](https://forum.qiime2.org/t/adding-ignore-missing-samples-argument/4215/2)
2. Related to issue #50
3. [Emperor API Docs](http://emperor.microbio.me/uno/build/html/generated/emperor.Emperor.html#emperor.Emperor)
4. [q2-sample-classifier docs (note the `missing-samples` parameter)](https://docs.qiime2.org/2018.6/plugins/available/sample-classifier/fit-classifier/)

**Comment by thermokarst** [comment]
Related to #50

**Comment by mortonjt** [comment]
yes! Or even better -- just have `ignore_missing_samples` to be true by default.  This saves the user a ton of trouble if they don't have to manually match samples in the metadata ( this is what gneiss is also doing by default).

**Comment by ElDeveloper** [comment]
Fixed in #71
